### PR TITLE
Create bootstrap.sh

### PR DESCRIPTION
**Pull request by Michael-Masas**
1st assignment - main script

### CODE REVIEW

**Comment by Michael-Masas** [comment]
@lioramilbaum
@greycat001

**Comment by lmilbaum** [comment]
Take a look at PR #1. Few of the comments are relevant to this one. Take a look.

File: Michael-Masas/bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+sudo yum install -y yum-utils;
+sudo yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo;
+sudo yum install -y docker-ce docker-ce-cli containerd.io;
```
Comment (lmilbaum) [inline Michael-Masas/bootstrap.sh:6]: You have have used a bit different package list than what Liroy have used. I would suggest that you both check what is the minimal amount of packages required.

File: Michael-Masas/bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+sudo yum install -y yum-utils;
```
Comment (lmilbaum) [inline Michael-Masas/bootstrap.sh:2]: Why every line is ended with ';'?

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: Michael-Masas/bootstrap.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    https://download.docker.com/linux/centos/docker-ce.repo;
+sudo yum install -y docker-ce docker-ce-cli containerd.io;
+sudo systemctl start docker;
+sudo systemctl enable docker;
+sudo yum install -y java-1.8.0-openjdk-devel;
+curl --silent --location http://pkg.jenkins-ci.org/redhat-stable/jenkins.repo | sudo tee /etc/yum.repos.d/jenkins.repo;
+sudo rpm --import https://pkg.jenkins.io/redhat-stable/jenkins.io.key;
+sudo yum install -y jenkins;
+sudo systemctl start jenkins;
+sudo systemctl enable jenkins;
```
Comment (lmilbaum) [inline Michael-Masas/bootstrap.sh:14]: It is more common to enable the service first and than start it.
BTW, Liroy should take a look. I think this issue was not covered by her.

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: Michael-Masas/bootstrap.sh (hunk 1 above)
Comment (Michael-Masas) [reply Michael-Masas/bootstrap.sh:6]: Those packages are from official Docker Docs , i don't see a reason to change it.

File: Michael-Masas/bootstrap.sh (hunk 3 above)
Comment (Michael-Masas) [reply Michael-Masas/bootstrap.sh:14]: i don't see an actual reason  to enable something that isn't running.

File: Michael-Masas/bootstrap.sh (hunk 2 above)
Comment (Michael-Masas) [reply Michael-Masas/bootstrap.sh:2]: to brake the commands.

Comment (Michael-Masas) [reply Michael-Masas/bootstrap.sh:2]: ack, removed.

**Review by Michael-Masas — COMMENTED** [review]
removed semicolons. 
script is working.
tested on centos7.

**Pull request merged**